src/Styling/Home.css

```diff
@@ -128,6 +126,80 @@
   width: 44%;
 }
 
+/* Project section */
```

Comment:
Even this CSS file is getting very large. I strongly recommend making the Project section it’s own react & css file.